src/organisms/gv-schema-form-control/gv-schema-form-control.js

```diff
@@ -213,8 +213,8 @@ export class GvSchemaFormControl extends UpdateAfterBrowser(LitElement) {
       element.type = 'password';
     }
 
-    if (this.control.description) {
-      element.description = this.control.description;
+    if (this.control.description || this.control.gioConfig?.banner) {
+      element.description = this.control.gioConfig?.banner?.text ?? this.control.description;
```

Comment:
Question: shouldn't we use this.control.description by default and banner as backup ?

Reply:
I asked myself this question. And I think that often the banner contains more information than the description/hint.
Otherwise you'd have to combine the 2 but I'm not sure